keyboards/keebio/iris/keymaps/ddone/rules.mk

```
@@ -0,0 +1,4 @@
+LTO_ENABLE = yes
+COMMAND_ENABLE = no
+MOUSEKEY_ENABLE = yes
+BOOTMAGIC_ENABLE = no
```

I would recommend: 
```suggestion
BOOTMAGIC_ENABLE = lite
```

This has none of the annoying hold key stuff, but can/will allow yo to hold a key to get into the bootloader.